### ISSUE TITLE
VR dependencies unavailability issues with `gradle dist`

### ISSUE THREAD

**Issue by rupeshkumar22** [processing/processing-android#718]
Suddenly with `gradle dist` It started showing the following errors. I remember it was working fine a few days back.

```
A problem occurred configuring project ':mode:libraries:vr'.
> Could not resolve all files for configuration ':mode:libraries:vr:implementationAar'.
   > Failed to transform sdk-audio-1.180.0.aar (com.google.vr:sdk-audio:1.180.0) to match attributes {artifactType=jar, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Could not find sdk-audio-1.180.0.aar (com.google.vr:sdk-audio:1.180.0).
        Searched in the following locations:
            https://repo.gradle.org/gradle/libs-releases/com/google/vr/sdk-audio/1.180.0/sdk-audio-1.180.0.aar
   > Failed to transform sdk-base-1.180.0.aar (com.google.vr:sdk-base:1.180.0) to match attributes {artifactType=jar, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Could not find sdk-base-1.180.0.aar (com.google.vr:sdk-base:1.180.0).
        Searched in the following locations:
            https://repo.gradle.org/gradle/libs-releases/com/google/vr/sdk-base/1.180.0/sdk-base-1.180.0.aar
   > Failed to transform sdk-base-1.180.0.aar (com.google.vr:sdk-base:1.180.0) to match attributes {artifactType=jar, org.gradle.status=release}.
      > Could not find sdk-base-1.180.0.aar (com.google.vr:sdk-base:1.180.0).
        Searched in the following locations:
            https://repo.gradle.org/gradle/libs-releases/com/google/vr/sdk-base/1.180.0/sdk-base-1.180.0.aar
   > Failed to transform sdk-common-1.180.0.aar (com.google.vr:sdk-common:1.180.0) to match attributes {artifactType=jar, org.gradle.status=release}.
      > Could not find sdk-common-1.180.0.aar (com.google.vr:sdk-common:1.180.0).
        Searched in the following locations:
            https://repo.gradle.org/gradle/libs-releases/com/google/vr/sdk-common/1.180.0/sdk-common-1.180.0.aar
 
```

Possible Solutions: We can place AAR file directly inside libs and use it with 
`implementation fileTree(dir: "libs", include: ["*.aar"])`

GVR releases can be found [here](https://github.com/googlevr/gvr-android-sdk/releases).

I tried using [jitpack for GVR ](https://jitpack.io/#googlevr/gvr-android-sdk) but the dependencies are still not getting resolved.

**Comment by rupeshkumar22** [comment]
Update: I tried today the master branch, `gradle dist` command still not working. It will start working once #713 is merged with the master after testing. Due to this issue, For nexus publish improvements I am working with the branch -
https://github.com/processing/processing-android/tree/sdkupdater-improvement-testing

**Comment by ranaaditya** [comment]
Hi @rupeshkumar22 Is this issue fixed or need some discussion with me ?

I have seen that Google has stopped the suport for the VR library and introduced new Cardboard API for VR. 

Two things cane be done here acc. to me: 
1. for temporary solution as of now we have to find the VR sdk online somewhere and have to add it in the processing final sdk as fat jar, but this will be the temporary solution for some time untill we will be migrated to the cardboard api.
2. Start migrating to Cardboard API, this will be hard to implement as it will need one more layer of processing APIover the cardboard api to make it work. This is the the final step which we have to take in future.
But for now we can see the point 1 untill we start migrating.

In case of any queries realted to this or any left over task I am open to discuss, just ping me on Email for scheduling the meeting.

Thanks !

**Comment by rupeshkumar22** [comment]
Hi @ranaaditya 
Yes, it is better to migrate to Cardboard API but it may require NDK not sure. I remember a comment by Andres on this. 
Yes, for now, I worked on including jars for VR, and it is working fine. Necessary Jars got merged into the project which is a temporary solution.
Thanks for highlighting this up,
Rupesh

**Comment by rupeshkumar22** [comment]
Fixed with a temporary solution([Added Jars](https://github.com/processing/processing-android/blob/c9163cb8f6ddc38bb30055a74ec9c105849238a4/mode/libraries/vr/libs/sdk-audio-1.180.0.aar))

Closing the issue, as a separate issue is there for the same - #723
